packages/patternfly-3/react-console/package.json

```diff
@@ -36,13 +36,14 @@
   "dependencies": {
     "@novnc/novnc": "^1.0.0",
     "classnames": "^2.2.5",
-    "patternfly": "^3.52.1",
-    "patternfly-react": "^2.10.0",
     "xterm": "^3.3.0"
   },
   "peerDependencies": {
     "prop-types": "^15.6.1",
     "react": "^16.3.2",
-    "react-dom": "^16.3.2"
+    "react-dom": "^16.3.2",
```

Comment:
If moving patternfly and patternfly-react to peerDependencies, we should also add them as devDependencies (per our discussion on yarn best practice). We could apply the same change to `patternfly-react-extensions` I suppose.

Reply:
Agree, done.